src/App.js

```diff
@@ -29,40 +29,65 @@ const App = () => {
 
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
-  const [squares, setSquares] = useState(generateSquares());
+  const [squareList, setSquares] = useState(generateSquares());
+  const [playCount, setPlayCount] = useState(0)
+  const [winner, setWinner] = useState('')
 
-  // Wave 2
-  // You will need to create a method to change the square 
-  //   When it is clicked on.
-  //   Then pass it into the squares as a callback
+  const callback = (updatedSquare) => {
```

Comment:
👍 , but I might use a more descriptive name to illustrate what this function does.